### DISCUSSION TITLE
✅ 🔗 Scheduled Popups (Duplicate)

### DISCUSSION THREAD

**Discussion by tekonemedia** [elementor/elementor#16988] [Feature Request]
With Black Friday again just around the corner, we find ourselves pulling our hair out preparing all our client's sites for the sales events. Would be great to have scheduled set date/time for running popups. Would save a lot of hassle for us developers as well as clients. would be spectacular to have integration with acf date picker for popup run times as well. This would make it a build it and forget it solution for client sites.

**Comment by tekonemedia** [comment]
Or even an option to turn popups on and off from within the popup template section.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new Date and Time range in the Advanced Rules of the Elementor Pro Popup in version 3.11! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
